### PR TITLE
Dataframe plot

### PR DESCRIPTION
**Pull request by joe-wolfe21**
closes: #32 

---

## What

Uses hvplot to adding basic dataframe plotting support

## How to Test

Build the docs and visit the new notebook. I committed with the notebook's output so that the example plots are rendered in the docs to give viewers a quick idea of the plotting functionality

### CODE REVIEW

File: rubicon/client/dataframe.py [hunk 1]
```diff
@@ -38,9 +41,36 @@ def _get_data(self):
...
+        For customizations, visit:
+        https://hvplot.holoviz.org/user_guide/Customization.html
+
+        Examples
+        --------
+        >>> # Log a line plot
+        >>> dataframe.plot(kind='line', x='Year', y='Number of Subscriptions')
+        """
+        # data is a dask dataframe
+        if sys.modules.get("dask") is not None:
```
Comment (ryanSoley) [inline rubicon/client/dataframe.py:65]: when would this ever be `None`?

**Review by ryanSoley — COMMENTED** [review]
I'm getting this warning when building the docs `/Users/nvd215/oss/rubicon/docs/source/plotting-support.ipynb:1: WARNING: Content block expected for the "raw" directive; none found.` and in the rendered output theres a block that says "Data type cannot be displayed:"

there's an empty cell at the end of the notebook - lets get rid of that

also the plot doesnt look great in dark mode, but thats probably more trouble than its worth

**Comment by ryanSoley** [comment]
oh and when i just view the notebook with `jupyter` I dont see the plot - not sure if thats causing any of the issues I mentioned. it renders in the docs when I build them fine

File: rubicon/client/dataframe.py (hunk 1 above)
Comment (joe-wolfe21) [reply rubicon/client/dataframe.py:65]: i added it as a precaution -- only if we refactored to select either `pandas` or `dask` on install, both are currently installed as part of the dependencies now

**Comment by joe-wolfe21** [comment]
> WARNING: Content block expected for the "raw" directive; none found.

Not sure why this is popping up. Based on our convo, decided to hide the warning of `"Data type cannot be displayed:` within the rendered output

File: docs/source/_static/custom.css [hunk 2]
```diff
@@ -20,4 +20,9 @@
     text-align: right;
     font-size: 20px;
     margin-top: -14px;
+}
+
+/* unknown warning was showing, manually hiding */
+#Default-Plotting-Support .admonition.warning {
+    display: none;
```
Comment (ryanSoley) [inline docs/source/_static/custom.css:27]: 🔥

**Review by ryanSoley — COMMENTED** [review]
last nitpick I promise - I don't love the title of the example. what about "Visualizing Logged Dataframes" just to be more specific

**Comment by joe-wolfe21** [comment]
@ryanSoley thanks for the solid PR feedback, it's ready for another look

**Review by ryanSoley — APPROVED** [review]
looks great!

**Pull request merged**